cmd/bundle.rb

```diff
@@ -255,6 +255,20 @@ def run
 
           _subcommand, *named_args = args.named
           Bundle::Commands::Add.run(*named_args, type: selected_types.first, global:, file:)
+        when "remove"
+          # We intentionally omit the `s` from `brews` and `casks` for ease of handling later.
```

Comment:
Would be nice to combine this logic with same for `add`

Reply:
Done!